std/experimental/ndslice/iteration.d

```diff
@@ -104,6 +104,8 @@ import std.meta;
 import std.experimental.ndslice.internal;
 import std.experimental.ndslice.slice; //: Slice;
 
+@trusted:
```

Comment:
@MetaLang's point was that it's a bad practice to annotate an entire module with `@trusted` as it can be very easy that something not safe slips through or will slip through in the future.